### DISCUSSION TITLE
Student Developer Pack Denied

### DISCUSSION THREAD

**Discussion by falchion10** [community/community#56973] [GitHub Education] [answered]
### Hi everyone,

I have been trying to apply for the Student Developer Pack for awhile and I keep getting a useless error and its frustrating as hell. I'm uploading my student ID with my graduation year, name, and school name on it. Can I please just get a manual review? The bot you guys use to automatically verify people is trash and needs an overhaul. Not to mention you don't even respond to emails anymore, you just force people to use the Github Community now. Please don't tell me the basic shit like "is the image clear?" or "does it have your school on it?".
<img width="326" alt="image" src="https://github.com/community/community/assets/122236745/139d5233-c727-4cc8-92d4-58b40d079386">

**Comment by falchion10** [comment, marked as answer]
Solved via someone else's solution.

**Comment by La-petite-baleine** [comment]
Hey, how was your case resolved? I keep getting denied despite having visible dates of my enrollment with my name clearly laid out. I have had 5 denials all in a span of the past hour. I don't know what to do at this point.
